dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Application/Authentication/AdaptiveCards/AdaptiveCardsAuthenticationBase.cs

```diff
@@ -13,7 +13,7 @@ internal abstract class AdaptiveCardsAuthenticationBase
         /// </summary>
         /// <param name="context">The turn context</param>
         /// <returns>The sign in response</returns>
-        public async Task<SignInResponse> AuthenticateAsync(ITurnContext context)
+        public Task<SignInResponse> AuthenticateAsync(ITurnContext context)
```

Comment:
could you undo this change as it's not related to the PR

Reply:
sure. I'll do that.

Reply:
Just FYI, this is producing a warning but I'll revert this change anyway.